spec/vehicle_locations.schema.json

```diff
@@ -0,0 +1,186 @@
+    "NaN",
+    ""
+  ],
+  "fields": [
+    {
+      "name": "date",
+      "type": "date",
+      "description": "Service date. References GTFS",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "timestamp",
+      "type": "time",
+      "description": "Recorded event time.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "location_ping_id",
+      "type": "string",
+      "description": "Identifies the recorded vehicle location event.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "trip_id_performed",
+      "type": "string",
+      "title": "ID referencing trips_performed.trip_id_performed",
+      "description": "Identifies the trip performed.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "stop_sequence",
+      "type": "string",
+      "title": "positive integer referencing stop_times.stop_sequence",
+      "description": "Ordered stop the vehicle is approaching or serving on a particular trip. References GTFS",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "vehicle_id",
+      "type": "string",
+      "title": "ID referencing vehicles.vehicle_id",
+      "description": "Identifies a vehicle.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "device_id",
+      "type": "string",
+      "title": "ID references devices.device_id",
+      "description": "Identifies the device that recorded the vehicle location. May be null if only a single device is reporting vehicle location and the device_id is not distinct from vehicle_id."
+    },
+    {
+      "name": "stop_id",
+      "type": "string",
+      "title": "ID referencing stops.stop_id",
+      "description": "Identifies the stop the vehicle is approaching or serving currently. References GTFS"
+    },
+    {
+      "name": "current_status",
+      "type": "integer",
+      "description": "Indicates the status of the vehicle in reference to a stop. Valid options are:\n0 - Incoming at.\n1 - Stopped at.\n2 - In transit to. References GTFS-Realtime",
+      "constraints": {
+        "enum": [
+          0,
+          1,
+          2
+        ]
+      }
+    },
+    {
+      "name": "latitude",
+      "type": "number",
+      "description": "Degrees North, in the WGS-84 coordinate system. References GTFS-Realtime"
+    },
+    {
+      "name": "longitude",
+      "type": "number",
+      "description": "Degrees East, in the WGS-84 coordinate system. References GTFS-Realtime"
+    },
+    {
+      "name": "gps_quality",
+      "type": "integer",
+      "description": "Indicates the quality of data and communication provided by the GPS. Valid options are:\n0 - Excellent.\n1 - Good.\n2 - Poor.",
+      "constraints": {
+        "enum": [
+          0,
+          1,
+          2
+        ]
+      }
+    },
+    {
+      "name": "bearing",
```

Comment:
The correct term is "heading", meaning what direction the vehicle is pointing relative to north, vs bearing, meaning the angle to a destination.